java-frontend/src/test/files/se/DivisionByZeroCheck.java

```diff
@@ -387,3 +387,17 @@ private void decodeBigInteger(int value) throws Exception {
     lowBits = -lowBits;
   }
 }
+
+class Assignment {
+  int myField;
+
+  public int calculate(int i) {
+    this.myField *= 0; // This is not a simple assignment and thus won't be correctly evaluated in member select.
```

Comment:
I don't get why it does not work. Even if it's not a simple assignment, it should work the same way as
```
public int calculate(int i) {
  myField *= 0;
  return i / myField; // DivByZero
}
```

As long as the stack contains the good symbolic value. may be related to wrong unstacking when considering that `this.field` is a simple assignment.